[2022-08-21 20:38:46,041] {processor.py:163} INFO - Started process (PID=86652) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:38:46,042] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:38:46,042] {logging_mixin.py:109} INFO - [2022-08-21 20:38:46,042] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:38:46,044] {logging_mixin.py:109} INFO - [2022-08-21 20:38:46,044] {dagbag.py:334} ERROR - Failed to import: /home/ubuntu/airflow/dags/dynamic_pricing.py
Traceback (most recent call last):
  File "/home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/dynamic_pricing.py", line 3, in <module>
    import request
ModuleNotFoundError: No module named 'request'
[2022-08-21 20:38:46,052] {processor.py:634} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:38:46,060] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.021 seconds
[2022-08-21 20:39:13,270] {processor.py:163} INFO - Started process (PID=86807) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:39:13,271] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:39:13,271] {logging_mixin.py:109} INFO - [2022-08-21 20:39:13,271] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:39:13,279] {logging_mixin.py:109} INFO - [2022-08-21 20:39:13,279] {dagbag.py:334} ERROR - Failed to import: /home/ubuntu/airflow/dags/dynamic_pricing.py
Traceback (most recent call last):
  File "/home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/dynamic_pricing.py", line 33, in <module>
    python_callable=print_world,
NameError: name 'print_world' is not defined
[2022-08-21 20:39:13,287] {processor.py:634} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:39:13,295] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.027 seconds
[2022-08-21 20:39:36,075] {processor.py:163} INFO - Started process (PID=86984) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:39:36,076] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:39:36,077] {logging_mixin.py:109} INFO - [2022-08-21 20:39:36,077] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:39:36,087] {logging_mixin.py:109} INFO - [2022-08-21 20:39:36,086] {dagbag.py:334} ERROR - Failed to import: /home/ubuntu/airflow/dags/dynamic_pricing.py
Traceback (most recent call last):
  File "/home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/dynamic_pricing.py", line 33, in <module>
    python_callable=print_world,
NameError: name 'print_world' is not defined
[2022-08-21 20:39:36,098] {processor.py:634} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:39:36,108] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.036 seconds
[2022-08-21 20:40:06,464] {processor.py:163} INFO - Started process (PID=87193) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:40:06,464] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:40:06,465] {logging_mixin.py:109} INFO - [2022-08-21 20:40:06,465] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:40:06,472] {logging_mixin.py:109} INFO - [2022-08-21 20:40:06,471] {dagbag.py:334} ERROR - Failed to import: /home/ubuntu/airflow/dags/dynamic_pricing.py
Traceback (most recent call last):
  File "/home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/dynamic_pricing.py", line 33, in <module>
    python_callable=print_world,
NameError: name 'print_world' is not defined
[2022-08-21 20:40:06,480] {processor.py:634} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:40:06,489] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.027 seconds
[2022-08-21 20:40:36,884] {processor.py:163} INFO - Started process (PID=87195) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:40:36,884] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:40:36,885] {logging_mixin.py:109} INFO - [2022-08-21 20:40:36,885] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:40:36,892] {logging_mixin.py:109} INFO - [2022-08-21 20:40:36,892] {dagbag.py:334} ERROR - Failed to import: /home/ubuntu/airflow/dags/dynamic_pricing.py
Traceback (most recent call last):
  File "/home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/dynamic_pricing.py", line 33, in <module>
    python_callable=print_world,
NameError: name 'print_world' is not defined
[2022-08-21 20:40:36,900] {processor.py:634} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:40:36,908] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.026 seconds
[2022-08-21 20:41:07,279] {processor.py:163} INFO - Started process (PID=87196) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:41:07,279] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:41:07,279] {logging_mixin.py:109} INFO - [2022-08-21 20:41:07,279] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:41:07,287] {logging_mixin.py:109} INFO - [2022-08-21 20:41:07,286] {dagbag.py:334} ERROR - Failed to import: /home/ubuntu/airflow/dags/dynamic_pricing.py
Traceback (most recent call last):
  File "/home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/dynamic_pricing.py", line 33, in <module>
    python_callable=print_world,
NameError: name 'print_world' is not defined
[2022-08-21 20:41:07,294] {processor.py:634} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:41:07,301] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.025 seconds
[2022-08-21 20:41:11,541] {processor.py:163} INFO - Started process (PID=87197) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:41:11,542] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:41:11,542] {logging_mixin.py:109} INFO - [2022-08-21 20:41:11,542] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:41:11,550] {logging_mixin.py:109} INFO - [2022-08-21 20:41:11,549] {dagbag.py:334} ERROR - Failed to import: /home/ubuntu/airflow/dags/dynamic_pricing.py
Traceback (most recent call last):
  File "/home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/dynamic_pricing.py", line 42, in <module>
    t1 >> t2
NameError: name 't1' is not defined
[2022-08-21 20:41:11,558] {processor.py:634} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:41:11,567] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.028 seconds
[2022-08-21 20:41:22,316] {processor.py:163} INFO - Started process (PID=87199) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:41:22,317] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:41:22,317] {logging_mixin.py:109} INFO - [2022-08-21 20:41:22,317] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:41:22,319] {logging_mixin.py:109} INFO - [2022-08-21 20:41:22,318] {dagbag.py:334} ERROR - Failed to import: /home/ubuntu/airflow/dags/dynamic_pricing.py
Traceback (most recent call last):
  File "/home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 846, in exec_module
  File "<frozen importlib._bootstrap_external>", line 983, in get_code
  File "<frozen importlib._bootstrap_external>", line 913, in source_to_code
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/dynamic_pricing.py", line 10
    def send_query(3.37.92.138:8001/dynamic_pricing/update) -> None:
                   ^
SyntaxError: invalid syntax
[2022-08-21 20:41:22,328] {processor.py:634} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:41:22,336] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.022 seconds
[2022-08-21 20:41:23,583] {processor.py:163} INFO - Started process (PID=87200) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:41:23,583] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:41:23,584] {logging_mixin.py:109} INFO - [2022-08-21 20:41:23,584] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:41:23,585] {logging_mixin.py:109} INFO - [2022-08-21 20:41:23,585] {dagbag.py:334} ERROR - Failed to import: /home/ubuntu/airflow/dags/dynamic_pricing.py
Traceback (most recent call last):
  File "/home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 846, in exec_module
  File "<frozen importlib._bootstrap_external>", line 983, in get_code
  File "<frozen importlib._bootstrap_external>", line 913, in source_to_code
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/dynamic_pricing.py", line 10
    def send_query(3.37.92.138:8001/dynamic_pricing/update) -> None:
                   ^
SyntaxError: invalid syntax
[2022-08-21 20:41:23,594] {processor.py:634} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:41:23,601] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.020 seconds
[2022-08-21 20:41:26,797] {processor.py:163} INFO - Started process (PID=87201) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:41:26,798] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:41:26,798] {logging_mixin.py:109} INFO - [2022-08-21 20:41:26,798] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:41:26,806] {logging_mixin.py:109} INFO - [2022-08-21 20:41:26,805] {dagbag.py:334} ERROR - Failed to import: /home/ubuntu/airflow/dags/dynamic_pricing.py
Traceback (most recent call last):
  File "/home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/dynamic_pricing.py", line 42, in <module>
    t1 >> t2
NameError: name 't1' is not defined
[2022-08-21 20:41:26,814] {processor.py:634} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:41:26,822] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.027 seconds
[2022-08-21 20:41:51,823] {processor.py:163} INFO - Started process (PID=87384) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:41:51,824] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:41:51,824] {logging_mixin.py:109} INFO - [2022-08-21 20:41:51,824] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:41:51,853] {logging_mixin.py:109} INFO - [2022-08-21 20:41:51,852] {dagbag.py:334} ERROR - Failed to import: /home/ubuntu/airflow/dags/dynamic_pricing.py
Traceback (most recent call last):
  File "/home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/dynamic_pricing.py", line 42, in <module>
    t1 >> t2
NameError: name 't1' is not defined
[2022-08-21 20:41:51,872] {processor.py:634} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:41:51,890] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.079 seconds
[2022-08-21 20:42:05,813] {processor.py:163} INFO - Started process (PID=87487) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:42:05,813] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:42:05,814] {logging_mixin.py:109} INFO - [2022-08-21 20:42:05,813] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:42:05,835] {processor.py:632} INFO - DAG(s) dict_keys(['dynamic_pricing']) retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:42:05,839] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:42:06,442] {logging_mixin.py:109} INFO - [2022-08-21 20:42:06,442] {manager.py:496} INFO - Created Permission View: can edit on DAG:dynamic_pricing
[2022-08-21 20:42:06,452] {logging_mixin.py:109} INFO - [2022-08-21 20:42:06,452] {manager.py:496} INFO - Created Permission View: can read on DAG:dynamic_pricing
[2022-08-21 20:42:06,452] {logging_mixin.py:109} INFO - [2022-08-21 20:42:06,452] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 20:42:06,460] {logging_mixin.py:109} INFO - [2022-08-21 20:42:06,460] {dag.py:2408} INFO - Creating ORM DAG for dynamic_pricing
[2022-08-21 20:42:06,465] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:42:06,471] {logging_mixin.py:109} INFO - [2022-08-21 20:42:06,471] {dag.py:2914} INFO - Setting next_dagrun for dynamic_pricing to 2022-08-20T00:00:00+00:00
[2022-08-21 20:42:06,574] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.764 seconds
[2022-08-21 20:42:12,499] {processor.py:163} INFO - Started process (PID=87545) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:42:12,499] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:42:12,499] {logging_mixin.py:109} INFO - [2022-08-21 20:42:12,499] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:42:12,517] {processor.py:632} INFO - DAG(s) dict_keys(['dynamic_pricing']) retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:42:12,521] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:42:12,523] {logging_mixin.py:109} INFO - [2022-08-21 20:42:12,523] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 20:42:12,533] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:42:12,540] {logging_mixin.py:109} INFO - [2022-08-21 20:42:12,540] {dag.py:2914} INFO - Setting next_dagrun for dynamic_pricing to 2022-08-20T00:00:00+00:00
[2022-08-21 20:42:12,551] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.054 seconds
[2022-08-21 20:42:43,062] {processor.py:163} INFO - Started process (PID=87858) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:42:43,062] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:42:43,063] {logging_mixin.py:109} INFO - [2022-08-21 20:42:43,063] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:42:43,081] {processor.py:632} INFO - DAG(s) dict_keys(['dynamic_pricing']) retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:42:43,085] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:42:43,092] {logging_mixin.py:109} INFO - [2022-08-21 20:42:43,092] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 20:42:43,102] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:42:43,108] {logging_mixin.py:109} INFO - [2022-08-21 20:42:43,108] {dag.py:2914} INFO - Setting next_dagrun for dynamic_pricing to 2022-08-20T17:00:00+00:00
[2022-08-21 20:42:43,119] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.060 seconds
[2022-08-21 20:43:14,027] {processor.py:163} INFO - Started process (PID=88132) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:43:14,027] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:43:14,028] {logging_mixin.py:109} INFO - [2022-08-21 20:43:14,028] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:43:14,046] {processor.py:632} INFO - DAG(s) dict_keys(['dynamic_pricing']) retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:43:14,050] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:43:14,057] {logging_mixin.py:109} INFO - [2022-08-21 20:43:14,057] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 20:43:14,069] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:43:14,075] {logging_mixin.py:109} INFO - [2022-08-21 20:43:14,075] {dag.py:2914} INFO - Setting next_dagrun for dynamic_pricing to 2022-08-21T07:00:00+00:00
[2022-08-21 20:43:14,085] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.060 seconds
[2022-08-21 20:43:44,428] {processor.py:163} INFO - Started process (PID=88314) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:43:44,428] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:43:44,429] {logging_mixin.py:109} INFO - [2022-08-21 20:43:44,429] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:43:44,447] {processor.py:632} INFO - DAG(s) dict_keys(['dynamic_pricing']) retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:43:44,451] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:43:44,459] {logging_mixin.py:109} INFO - [2022-08-21 20:43:44,459] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 20:43:44,469] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:43:44,474] {logging_mixin.py:109} INFO - [2022-08-21 20:43:44,474] {dag.py:2914} INFO - Setting next_dagrun for dynamic_pricing to 2022-08-21T07:00:00+00:00
[2022-08-21 20:43:44,485] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.059 seconds
[2022-08-21 20:44:00,839] {processor.py:163} INFO - Started process (PID=88435) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:44:00,839] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:44:00,840] {logging_mixin.py:109} INFO - [2022-08-21 20:44:00,840] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:44:00,860] {processor.py:632} INFO - DAG(s) dict_keys(['dynamic_pricing']) retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:44:00,864] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:44:01,114] {logging_mixin.py:109} INFO - [2022-08-21 20:44:01,114] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 20:44:01,125] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:44:01,130] {logging_mixin.py:109} INFO - [2022-08-21 20:44:01,130] {dag.py:2914} INFO - Setting next_dagrun for dynamic_pricing to 2022-08-21T07:00:00+00:00
[2022-08-21 20:44:01,144] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.307 seconds
[2022-08-21 20:44:08,323] {processor.py:163} INFO - Started process (PID=88502) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:44:08,324] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:44:08,324] {logging_mixin.py:109} INFO - [2022-08-21 20:44:08,324] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:44:08,342] {processor.py:632} INFO - DAG(s) dict_keys(['dynamic_pricing']) retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:44:08,346] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:44:08,348] {logging_mixin.py:109} INFO - [2022-08-21 20:44:08,348] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 20:44:08,358] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:44:08,365] {logging_mixin.py:109} INFO - [2022-08-21 20:44:08,365] {dag.py:2914} INFO - Setting next_dagrun for dynamic_pricing to 2022-08-21T07:00:00+00:00
[2022-08-21 20:44:08,376] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.055 seconds
[2022-08-21 20:44:38,463] {processor.py:163} INFO - Started process (PID=88768) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:44:38,464] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:44:38,464] {logging_mixin.py:109} INFO - [2022-08-21 20:44:38,464] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:44:38,483] {processor.py:632} INFO - DAG(s) dict_keys(['dynamic_pricing']) retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:44:38,487] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:44:38,494] {logging_mixin.py:109} INFO - [2022-08-21 20:44:38,493] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 20:44:38,504] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:44:38,510] {logging_mixin.py:109} INFO - [2022-08-21 20:44:38,510] {dag.py:2914} INFO - Setting next_dagrun for dynamic_pricing to 2022-08-21T07:00:00+00:00
[2022-08-21 20:44:38,521] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.059 seconds
[2022-08-21 20:45:08,959] {processor.py:163} INFO - Started process (PID=88981) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:45:08,960] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:45:08,960] {logging_mixin.py:109} INFO - [2022-08-21 20:45:08,960] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:45:08,980] {processor.py:632} INFO - DAG(s) dict_keys(['dynamic_pricing']) retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:45:08,984] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:45:08,990] {logging_mixin.py:109} INFO - [2022-08-21 20:45:08,990] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 20:45:09,001] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:45:09,007] {logging_mixin.py:109} INFO - [2022-08-21 20:45:09,007] {dag.py:2914} INFO - Setting next_dagrun for dynamic_pricing to 2022-08-21T07:00:00+00:00
[2022-08-21 20:45:09,019] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.062 seconds
[2022-08-21 20:45:39,453] {processor.py:163} INFO - Started process (PID=89014) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:45:39,454] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:45:39,455] {logging_mixin.py:109} INFO - [2022-08-21 20:45:39,454] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:45:39,473] {processor.py:632} INFO - DAG(s) dict_keys(['dynamic_pricing']) retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:45:39,477] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:45:39,486] {logging_mixin.py:109} INFO - [2022-08-21 20:45:39,485] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 20:45:39,495] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:45:39,501] {logging_mixin.py:109} INFO - [2022-08-21 20:45:39,500] {dag.py:2914} INFO - Setting next_dagrun for dynamic_pricing to 2022-08-21T07:00:00+00:00
[2022-08-21 20:45:39,511] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.059 seconds
[2022-08-21 20:46:09,865] {processor.py:163} INFO - Started process (PID=89075) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:46:09,865] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:46:09,865] {logging_mixin.py:109} INFO - [2022-08-21 20:46:09,865] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:46:09,886] {processor.py:632} INFO - DAG(s) dict_keys(['dynamic_pricing']) retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:46:09,890] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:46:09,896] {logging_mixin.py:109} INFO - [2022-08-21 20:46:09,896] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 20:46:09,906] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:46:09,911] {logging_mixin.py:109} INFO - [2022-08-21 20:46:09,911] {dag.py:2914} INFO - Setting next_dagrun for dynamic_pricing to 2022-08-21T07:00:00+00:00
[2022-08-21 20:46:09,923] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.060 seconds
[2022-08-21 20:46:40,314] {processor.py:163} INFO - Started process (PID=89107) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:46:40,315] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:46:40,315] {logging_mixin.py:109} INFO - [2022-08-21 20:46:40,315] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:46:40,336] {processor.py:632} INFO - DAG(s) dict_keys(['dynamic_pricing']) retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:46:40,340] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:46:40,348] {logging_mixin.py:109} INFO - [2022-08-21 20:46:40,348] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 20:46:40,358] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:46:40,365] {logging_mixin.py:109} INFO - [2022-08-21 20:46:40,364] {dag.py:2914} INFO - Setting next_dagrun for dynamic_pricing to 2022-08-21T07:00:00+00:00
[2022-08-21 20:46:40,376] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.064 seconds
[2022-08-21 20:47:10,819] {processor.py:163} INFO - Started process (PID=89108) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:47:10,819] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:47:10,820] {logging_mixin.py:109} INFO - [2022-08-21 20:47:10,820] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:47:10,840] {processor.py:632} INFO - DAG(s) dict_keys(['dynamic_pricing']) retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:47:10,845] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:47:10,854] {logging_mixin.py:109} INFO - [2022-08-21 20:47:10,854] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 20:47:10,863] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:47:10,869] {logging_mixin.py:109} INFO - [2022-08-21 20:47:10,869] {dag.py:2914} INFO - Setting next_dagrun for dynamic_pricing to 2022-08-21T07:00:00+00:00
[2022-08-21 20:47:10,880] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.063 seconds
[2022-08-21 20:47:41,257] {processor.py:163} INFO - Started process (PID=89203) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:47:41,258] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:47:41,258] {logging_mixin.py:109} INFO - [2022-08-21 20:47:41,258] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:47:41,278] {processor.py:632} INFO - DAG(s) dict_keys(['dynamic_pricing']) retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:47:41,282] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:47:41,289] {logging_mixin.py:109} INFO - [2022-08-21 20:47:41,289] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 20:47:41,299] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:47:41,304] {logging_mixin.py:109} INFO - [2022-08-21 20:47:41,304] {dag.py:2914} INFO - Setting next_dagrun for dynamic_pricing to 2022-08-21T07:00:00+00:00
[2022-08-21 20:47:41,315] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.061 seconds
[2022-08-21 20:48:11,385] {processor.py:163} INFO - Started process (PID=89426) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:48:11,385] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:48:11,386] {logging_mixin.py:109} INFO - [2022-08-21 20:48:11,386] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:48:11,405] {processor.py:632} INFO - DAG(s) dict_keys(['dynamic_pricing']) retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:48:11,409] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:48:11,415] {logging_mixin.py:109} INFO - [2022-08-21 20:48:11,415] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 20:48:11,425] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:48:11,431] {logging_mixin.py:109} INFO - [2022-08-21 20:48:11,431] {dag.py:2914} INFO - Setting next_dagrun for dynamic_pricing to 2022-08-21T07:00:00+00:00
[2022-08-21 20:48:11,442] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.059 seconds
[2022-08-21 20:48:42,166] {processor.py:163} INFO - Started process (PID=89628) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:48:42,167] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:48:42,167] {logging_mixin.py:109} INFO - [2022-08-21 20:48:42,167] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:48:42,186] {processor.py:632} INFO - DAG(s) dict_keys(['dynamic_pricing']) retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:48:42,190] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:48:42,196] {logging_mixin.py:109} INFO - [2022-08-21 20:48:42,196] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 20:48:42,206] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:48:42,211] {logging_mixin.py:109} INFO - [2022-08-21 20:48:42,211] {dag.py:2914} INFO - Setting next_dagrun for dynamic_pricing to 2022-08-21T07:00:00+00:00
[2022-08-21 20:48:42,222] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.058 seconds
[2022-08-21 20:49:12,682] {processor.py:163} INFO - Started process (PID=89830) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:49:12,683] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:49:12,683] {logging_mixin.py:109} INFO - [2022-08-21 20:49:12,683] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:49:12,703] {processor.py:632} INFO - DAG(s) dict_keys(['dynamic_pricing']) retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:49:12,707] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:49:12,714] {logging_mixin.py:109} INFO - [2022-08-21 20:49:12,714] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 20:49:12,724] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:49:12,729] {logging_mixin.py:109} INFO - [2022-08-21 20:49:12,729] {dag.py:2914} INFO - Setting next_dagrun for dynamic_pricing to 2022-08-21T07:00:00+00:00
[2022-08-21 20:49:12,740] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.060 seconds
[2022-08-21 20:49:42,935] {processor.py:163} INFO - Started process (PID=90003) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:49:42,935] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:49:42,936] {logging_mixin.py:109} INFO - [2022-08-21 20:49:42,936] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:49:42,954] {processor.py:632} INFO - DAG(s) dict_keys(['dynamic_pricing']) retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:49:42,958] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:49:42,965] {logging_mixin.py:109} INFO - [2022-08-21 20:49:42,965] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 20:49:42,974] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:49:42,980] {logging_mixin.py:109} INFO - [2022-08-21 20:49:42,980] {dag.py:2914} INFO - Setting next_dagrun for dynamic_pricing to 2022-08-21T07:00:00+00:00
[2022-08-21 20:49:42,991] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.058 seconds
[2022-08-21 20:50:13,456] {processor.py:163} INFO - Started process (PID=90154) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:50:13,456] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:50:13,456] {logging_mixin.py:109} INFO - [2022-08-21 20:50:13,456] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:50:13,479] {processor.py:632} INFO - DAG(s) dict_keys(['dynamic_pricing']) retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:50:13,483] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:50:13,489] {logging_mixin.py:109} INFO - [2022-08-21 20:50:13,489] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 20:50:13,499] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:50:13,505] {logging_mixin.py:109} INFO - [2022-08-21 20:50:13,504] {dag.py:2914} INFO - Setting next_dagrun for dynamic_pricing to 2022-08-21T07:00:00+00:00
[2022-08-21 20:50:13,516] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.063 seconds
[2022-08-21 20:50:43,768] {processor.py:163} INFO - Started process (PID=90186) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:50:43,769] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:50:43,769] {logging_mixin.py:109} INFO - [2022-08-21 20:50:43,769] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:50:43,788] {processor.py:632} INFO - DAG(s) dict_keys(['dynamic_pricing']) retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:50:43,792] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:50:43,798] {logging_mixin.py:109} INFO - [2022-08-21 20:50:43,798] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 20:50:43,808] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:50:43,813] {logging_mixin.py:109} INFO - [2022-08-21 20:50:43,813] {dag.py:2914} INFO - Setting next_dagrun for dynamic_pricing to 2022-08-21T07:00:00+00:00
[2022-08-21 20:50:43,824] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.058 seconds
[2022-08-21 20:51:14,265] {processor.py:163} INFO - Started process (PID=90217) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:51:14,266] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:51:14,266] {logging_mixin.py:109} INFO - [2022-08-21 20:51:14,266] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:51:14,285] {processor.py:632} INFO - DAG(s) dict_keys(['dynamic_pricing']) retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:51:14,289] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:51:14,296] {logging_mixin.py:109} INFO - [2022-08-21 20:51:14,296] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 20:51:14,305] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:51:14,311] {logging_mixin.py:109} INFO - [2022-08-21 20:51:14,311] {dag.py:2914} INFO - Setting next_dagrun for dynamic_pricing to 2022-08-21T07:00:00+00:00
[2022-08-21 20:51:14,322] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.058 seconds
[2022-08-21 20:51:33,502] {processor.py:163} INFO - Started process (PID=90250) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:51:33,502] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:51:33,503] {logging_mixin.py:109} INFO - [2022-08-21 20:51:33,503] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:51:33,523] {processor.py:632} INFO - DAG(s) dict_keys(['dynamic_pricing']) retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:51:33,526] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:51:33,779] {logging_mixin.py:109} INFO - [2022-08-21 20:51:33,779] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 20:51:33,788] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:51:33,794] {logging_mixin.py:109} INFO - [2022-08-21 20:51:33,793] {dag.py:2914} INFO - Setting next_dagrun for dynamic_pricing to 2022-08-21T07:00:00+00:00
[2022-08-21 20:51:33,807] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.307 seconds
[2022-08-21 20:51:49,571] {processor.py:163} INFO - Started process (PID=90402) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:51:49,572] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:51:49,572] {logging_mixin.py:109} INFO - [2022-08-21 20:51:49,572] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:51:49,600] {processor.py:632} INFO - DAG(s) dict_keys(['dynamic_pricing']) retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:51:49,605] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:51:49,608] {logging_mixin.py:109} INFO - [2022-08-21 20:51:49,608] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 20:51:49,622] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:51:49,631] {logging_mixin.py:109} INFO - [2022-08-21 20:51:49,631] {dag.py:2914} INFO - Setting next_dagrun for dynamic_pricing to 2022-08-21T07:00:00+00:00
[2022-08-21 20:51:49,645] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.077 seconds
[2022-08-21 20:52:19,701] {processor.py:163} INFO - Started process (PID=90674) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:52:19,702] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:52:19,702] {logging_mixin.py:109} INFO - [2022-08-21 20:52:19,702] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:52:19,721] {processor.py:632} INFO - DAG(s) dict_keys(['dynamic_pricing']) retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:52:19,725] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:52:19,732] {logging_mixin.py:109} INFO - [2022-08-21 20:52:19,732] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 20:52:19,742] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:52:19,748] {logging_mixin.py:109} INFO - [2022-08-21 20:52:19,748] {dag.py:2914} INFO - Setting next_dagrun for dynamic_pricing to 2022-08-21T07:00:00+00:00
[2022-08-21 20:52:19,759] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.060 seconds
[2022-08-21 20:52:50,254] {processor.py:163} INFO - Started process (PID=90801) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:52:50,255] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:52:50,255] {logging_mixin.py:109} INFO - [2022-08-21 20:52:50,255] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:52:50,275] {processor.py:632} INFO - DAG(s) dict_keys(['dynamic_pricing']) retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:52:50,279] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:52:50,287] {logging_mixin.py:109} INFO - [2022-08-21 20:52:50,286] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 20:52:50,298] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:52:50,305] {logging_mixin.py:109} INFO - [2022-08-21 20:52:50,304] {dag.py:2914} INFO - Setting next_dagrun for dynamic_pricing to 2022-08-21T07:00:00+00:00
[2022-08-21 20:52:50,317] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.065 seconds
[2022-08-21 20:53:20,376] {processor.py:163} INFO - Started process (PID=90862) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:53:20,377] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:53:20,377] {logging_mixin.py:109} INFO - [2022-08-21 20:53:20,377] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:53:20,397] {processor.py:632} INFO - DAG(s) dict_keys(['dynamic_pricing']) retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:53:20,401] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:53:20,410] {logging_mixin.py:109} INFO - [2022-08-21 20:53:20,410] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 20:53:20,420] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:53:20,426] {logging_mixin.py:109} INFO - [2022-08-21 20:53:20,426] {dag.py:2914} INFO - Setting next_dagrun for dynamic_pricing to 2022-08-21T07:00:00+00:00
[2022-08-21 20:53:20,438] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.064 seconds
[2022-08-21 20:53:51,235] {processor.py:163} INFO - Started process (PID=90865) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:53:51,236] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:53:51,236] {logging_mixin.py:109} INFO - [2022-08-21 20:53:51,236] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:53:51,257] {processor.py:632} INFO - DAG(s) dict_keys(['dynamic_pricing']) retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:53:51,261] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:53:51,268] {logging_mixin.py:109} INFO - [2022-08-21 20:53:51,268] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 20:53:51,279] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:53:51,285] {logging_mixin.py:109} INFO - [2022-08-21 20:53:51,285] {dag.py:2914} INFO - Setting next_dagrun for dynamic_pricing to 2022-08-21T07:00:00+00:00
[2022-08-21 20:53:51,296] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.063 seconds
[2022-08-21 20:54:21,377] {processor.py:163} INFO - Started process (PID=90866) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:54:21,377] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:54:21,377] {logging_mixin.py:109} INFO - [2022-08-21 20:54:21,377] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:54:21,398] {processor.py:632} INFO - DAG(s) dict_keys(['dynamic_pricing']) retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:54:21,402] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:54:21,410] {logging_mixin.py:109} INFO - [2022-08-21 20:54:21,410] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 20:54:21,420] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:54:21,426] {logging_mixin.py:109} INFO - [2022-08-21 20:54:21,426] {dag.py:2914} INFO - Setting next_dagrun for dynamic_pricing to 2022-08-21T07:00:00+00:00
[2022-08-21 20:54:21,439] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.064 seconds
[2022-08-21 20:54:52,216] {processor.py:163} INFO - Started process (PID=90942) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:54:52,216] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:54:52,217] {logging_mixin.py:109} INFO - [2022-08-21 20:54:52,217] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:54:52,238] {processor.py:632} INFO - DAG(s) dict_keys(['dynamic_pricing']) retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:54:52,242] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:54:52,249] {logging_mixin.py:109} INFO - [2022-08-21 20:54:52,249] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 20:54:52,259] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:54:52,264] {logging_mixin.py:109} INFO - [2022-08-21 20:54:52,264] {dag.py:2914} INFO - Setting next_dagrun for dynamic_pricing to 2022-08-21T07:00:00+00:00
[2022-08-21 20:54:52,275] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.062 seconds
[2022-08-21 20:55:22,692] {processor.py:163} INFO - Started process (PID=91034) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:55:22,692] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:55:22,692] {logging_mixin.py:109} INFO - [2022-08-21 20:55:22,692] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:55:22,712] {processor.py:632} INFO - DAG(s) dict_keys(['dynamic_pricing']) retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:55:22,716] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:55:22,723] {logging_mixin.py:109} INFO - [2022-08-21 20:55:22,723] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 20:55:22,733] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:55:22,739] {logging_mixin.py:109} INFO - [2022-08-21 20:55:22,739] {dag.py:2914} INFO - Setting next_dagrun for dynamic_pricing to 2022-08-21T07:00:00+00:00
[2022-08-21 20:55:22,749] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.060 seconds
[2022-08-21 20:55:41,079] {processor.py:163} INFO - Started process (PID=91141) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:55:41,080] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:55:41,080] {logging_mixin.py:109} INFO - [2022-08-21 20:55:41,080] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:55:41,101] {processor.py:632} INFO - DAG(s) dict_keys(['dynamic_pricing']) retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:55:41,105] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:55:41,112] {logging_mixin.py:109} INFO - [2022-08-21 20:55:41,112] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 20:55:41,121] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:55:41,127] {logging_mixin.py:109} INFO - [2022-08-21 20:55:41,127] {dag.py:2914} INFO - Setting next_dagrun for dynamic_pricing to 2022-08-21T07:00:00+00:00
[2022-08-21 20:55:41,140] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.062 seconds
[2022-08-21 20:56:12,364] {processor.py:163} INFO - Started process (PID=91167) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:56:12,364] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:56:12,365] {logging_mixin.py:109} INFO - [2022-08-21 20:56:12,365] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:56:12,384] {processor.py:632} INFO - DAG(s) dict_keys(['dynamic_pricing']) retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:56:12,388] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:56:12,394] {logging_mixin.py:109} INFO - [2022-08-21 20:56:12,394] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 20:56:12,404] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:56:12,409] {logging_mixin.py:109} INFO - [2022-08-21 20:56:12,409] {dag.py:2914} INFO - Setting next_dagrun for dynamic_pricing to 2022-08-21T07:00:00+00:00
[2022-08-21 20:56:12,419] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.058 seconds
[2022-08-21 20:56:43,546] {processor.py:163} INFO - Started process (PID=91171) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:56:43,547] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:56:43,547] {logging_mixin.py:109} INFO - [2022-08-21 20:56:43,547] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:56:43,566] {processor.py:632} INFO - DAG(s) dict_keys(['dynamic_pricing']) retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:56:43,570] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:56:43,577] {logging_mixin.py:109} INFO - [2022-08-21 20:56:43,576] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 20:56:43,586] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:56:43,592] {logging_mixin.py:109} INFO - [2022-08-21 20:56:43,592] {dag.py:2914} INFO - Setting next_dagrun for dynamic_pricing to 2022-08-21T07:00:00+00:00
[2022-08-21 20:56:43,602] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.058 seconds
[2022-08-21 20:56:48,748] {processor.py:163} INFO - Started process (PID=91173) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:56:48,748] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:56:48,749] {logging_mixin.py:109} INFO - [2022-08-21 20:56:48,749] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:56:48,768] {processor.py:632} INFO - DAG(s) dict_keys(['dynamic_pricing']) retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:56:48,772] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:56:48,780] {logging_mixin.py:109} INFO - [2022-08-21 20:56:48,780] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 20:56:48,789] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:56:48,795] {logging_mixin.py:109} INFO - [2022-08-21 20:56:48,795] {dag.py:2914} INFO - Setting next_dagrun for dynamic_pricing to 2022-08-21T07:00:00+00:00
[2022-08-21 20:56:48,807] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.061 seconds
[2022-08-21 20:57:09,189] {processor.py:163} INFO - Started process (PID=91310) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:57:09,189] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:57:09,189] {logging_mixin.py:109} INFO - [2022-08-21 20:57:09,189] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:57:09,208] {processor.py:632} INFO - DAG(s) dict_keys(['dynamic_pricing']) retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:57:09,212] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:57:09,218] {logging_mixin.py:109} INFO - [2022-08-21 20:57:09,218] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 20:57:09,228] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:57:09,235] {logging_mixin.py:109} INFO - [2022-08-21 20:57:09,235] {dag.py:2914} INFO - Setting next_dagrun for dynamic_pricing to 2022-08-21T07:00:00+00:00
[2022-08-21 20:57:09,245] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.058 seconds
[2022-08-21 20:57:40,963] {processor.py:163} INFO - Started process (PID=91645) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:57:40,963] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:57:40,964] {logging_mixin.py:109} INFO - [2022-08-21 20:57:40,964] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:57:40,982] {processor.py:632} INFO - DAG(s) dict_keys(['dynamic_pricing']) retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:57:40,986] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:57:40,992] {logging_mixin.py:109} INFO - [2022-08-21 20:57:40,992] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 20:57:41,003] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:57:41,008] {logging_mixin.py:109} INFO - [2022-08-21 20:57:41,008] {dag.py:2914} INFO - Setting next_dagrun for dynamic_pricing to 2022-08-21T07:00:00+00:00
[2022-08-21 20:57:41,020] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.059 seconds
[2022-08-21 20:58:15,912] {processor.py:163} INFO - Started process (PID=92126) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:58:15,913] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:58:15,913] {logging_mixin.py:109} INFO - [2022-08-21 20:58:15,913] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:58:15,938] {processor.py:632} INFO - DAG(s) dict_keys(['dynamic_pricing']) retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:58:15,942] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:58:15,949] {logging_mixin.py:109} INFO - [2022-08-21 20:58:15,949] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 20:58:15,961] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:58:15,967] {logging_mixin.py:109} INFO - [2022-08-21 20:58:15,967] {dag.py:2914} INFO - Setting next_dagrun for dynamic_pricing to 2022-08-21T07:03:00+00:00
[2022-08-21 20:58:15,986] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.077 seconds
[2022-08-21 20:58:46,278] {processor.py:163} INFO - Started process (PID=92574) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:58:46,279] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:58:46,279] {logging_mixin.py:109} INFO - [2022-08-21 20:58:46,279] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:58:46,298] {processor.py:632} INFO - DAG(s) dict_keys(['dynamic_pricing']) retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:58:46,302] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:58:46,310] {logging_mixin.py:109} INFO - [2022-08-21 20:58:46,310] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 20:58:46,320] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:58:46,325] {logging_mixin.py:109} INFO - [2022-08-21 20:58:46,325] {dag.py:2914} INFO - Setting next_dagrun for dynamic_pricing to 2022-08-21T07:09:00+00:00
[2022-08-21 20:58:46,335] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.059 seconds
[2022-08-21 20:59:17,452] {processor.py:163} INFO - Started process (PID=93041) to work on /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:59:17,452] {processor.py:620} INFO - Processing file /home/ubuntu/airflow/dags/dynamic_pricing.py for tasks to queue
[2022-08-21 20:59:17,453] {logging_mixin.py:109} INFO - [2022-08-21 20:59:17,453] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:59:17,472] {processor.py:632} INFO - DAG(s) dict_keys(['dynamic_pricing']) retrieved from /home/ubuntu/airflow/dags/dynamic_pricing.py
[2022-08-21 20:59:17,475] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:59:17,484] {logging_mixin.py:109} INFO - [2022-08-21 20:59:17,484] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 20:59:17,493] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 20:59:17,499] {logging_mixin.py:109} INFO - [2022-08-21 20:59:17,499] {dag.py:2914} INFO - Setting next_dagrun for dynamic_pricing to 2022-08-21T07:15:00+00:00
[2022-08-21 20:59:17,510] {processor.py:171} INFO - Processing /home/ubuntu/airflow/dags/dynamic_pricing.py took 0.061 seconds
